templates/flake-parts/.envrc

```diff
@@ -5,7 +5,8 @@ fi
 nix_direnv_watch_file devenv.nix
 nix_direnv_watch_file devenv.lock
 nix_direnv_watch_file devenv.yaml
-if ! use flake . --impure
+
+if ! use flake . --override-input devenv-root "file+file:"<(pwd)
```

Comment:
this doesn't work for me (nix 2.18.1 aarch64-darwin) - needs to be `"file+file://"<(pwd)`